api/v1alpha1/proxmoxmachine_types.go

```diff
@@ -115,12 +115,13 @@ type ProxmoxMachineSpec struct {
 	MetadataSettings *MetadataSettings `json:"metadataSettings,omitempty"`
 
 	// AllowedNodes specifies all Proxmox nodes which will be considered
-	// for operations. This implies that VMs can be cloned on different nodes from
-	// the node which holds the VM template.
+	// for operations. This implies that VMs can be cloned on different nodes.
 	//
 	// This field is optional and should only be set if you want to restrict
 	// the nodes where the VM can be cloned.
-	// If not set, the ProxmoxCluster will be used to determine the nodes.
+	//
+	// if set overrides ProxmoxCluster.spec.AllowedNodes
```

Comment:
I would prefer to reject a config that has both rather than overriding. Overriding behaviours can be confusing.

Reply:
ok understood.